include/lsst/afw/table/Simple.h

```diff
@@ -205,7 +206,7 @@ inline void SimpleRecord::setId(RecordId id) { set(SimpleTable::getIdKey(), id);
 
 inline IcrsCoord SimpleRecord::getCoord() const { return get(SimpleTable::getCoordKey()); }
 inline void SimpleRecord::setCoord(IcrsCoord const & coord) { set(SimpleTable::getCoordKey(), coord); }
-inline void SimpleRecord::setCoord(Coord const & coord) { set(SimpleTable::getCoordKey(), coord); }
+inline void SimpleRecord::setCoord(Coord const & coord) { SimpleTable::getCoordKey().set(*this, coord); }
 
```

Comment:
This is a preexisting bug I discovered on this issue and worked around but should have reported.  It's now https://jira.lsstcorp.org/browse/DM-2872.